config/initializers/figaro.rb

```diff
@@ -6,7 +6,6 @@
   'domain_name',
   'enable_identity_verification',
   'enable_test_routes',
-  'equifax_ssh_passphrase',
```

Comment:
Works for me! I haven't worked with a release branch process like this before so I'm not sure what the pitfalls will really be vs what I imagine they could be. Thanks!